doc/man3/STORE_INFO.pod

```diff
@@ -0,0 +1,146 @@
+=pod
+
+=head1 NAME
+
+STORE_INFO, STORE_INFO_get_type, STORE_INFO_get0_NAME,
+STORE_INFO_get0_PARAMS, STORE_INFO_get0_PKEY, STORE_INFO_get0_CERT,
+STORE_INFO_get0_CRL, STORE_INFO_type_string, STORE_INFO_free,
+STORE_INFO_new_NAME, STORE_INFO_new_PARAMS, STORE_INFO_new_PKEY,
+STORE_INFO_new_CERT, STORE_INFO_new_CRL, STORE_INFO_new_ENDOFDATA
+- Functions to manipulate STORE_INFO objects
+
+=head1 SYNOPSIS
+
+ #include <openssl/store.h>
+ 
+ typedef struct store_info_st STORE_INFO;
+
+ int STORE_INFO_get_type(const STORE_INFO *store_info);
+ const char *STORE_INFO_get0_NAME(const STORE_INFO *store_info);
+ DSA *STORE_INFO_get0_PARAMS(const STORE_INFO *store_info);
+ EVP_PKEY *STORE_INFO_get0_PKEY(const STORE_INFO *store_info);
+ X509 *STORE_INFO_get0_CERT(const STORE_INFO *store_info);
+ X509_CRL *STORE_INFO_get0_CRL(const STORE_INFO *store_info);
+
+ const char *STORE_INFO_type_string(int type);
+
+ void STORE_INFO_free(STORE_INFO *store_info);
+
+ STORE_INFO *STORE_INFO_new_NAME(char *name);
+ STORE_INFO *STORE_INFO_new_PARAMS(DSA *dsa_params);
+ STORE_INFO *STORE_INFO_new_PKEY(EVP_PKEY *pkey);
+ STORE_INFO *STORE_INFO_new_CERT(X509 *x509);
+ STORE_INFO *STORE_INFO_new_CRL(X509_CRL *crl);
+
+ STORE_INFO *STORE_INFO_new_ENDOFDATA(void);
+
+=head1 DESCRIPTION
+
+These functions are primarly useful for applications to retrieve
+supported objects from B<STORE_INFO> objects and for scheme specific
+loaders to create B<STORE_INFO> holders.
+
+=head2 Types
+
+B<STORE_INFO> is an opaque type that's just an intermediary holder for
+the objects that have been retrieved by STORE_load() and similar
+functions.
+Supported OpenSSL type object can be extracted using one of
+STORE_INFO_get0_TYPE(), after which the B<STORE_INFO> should be freed
+with STORE_INFO_free().
```

Comment:
That should be rephrased, and the corresponding `_get1_` functions should be documented as well.  Those do allow using the extracted object after freeing the `STORE_INFO`